content/dev-tools/faucets.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Maybe put this file into a `data` folder or something

Reply:
I think it's useful to keep the data near where it's used in this case since it's super related to the test & there's not many files so it doesn't clutter a reader's ability to tell what all the files are for / doing.